packages/visx-demo/package.json

```diff
@@ -87,7 +87,7 @@
     "nprogress": "^0.2.0",
     "prismjs": "^1.19.0",
     "raw-loader": "^4.0.0",
-    "react": "^16.9.0",
+    "react": "^16.9.0 || ^17.0.0-0",
```

Comment:
since this one is a `dependency` (and not really intended to be installed) we could probably just use `^17.0.0` here? this would also let us test and make sure everything works with `17` in the demo site.